site/content/docs/5.3/components/card.md

```diff
@@ -12,6 +12,18 @@ toc: true
 
 A **card** is a flexible and extensible content container. It includes options for headers and footers, a wide variety of content, contextual background colors, and powerful display options. If you're familiar with Boosted 3, cards replace our old panels, wells, and thumbnails. Similar functionality to those components is available as modifier classes for cards.
 
+{{< design-callout-alert info >}}
+**On the Orange Design System website**, you'll find the [Cards](https://system.design.orange.com/0c1af118d/p/82eaf1-cards/b/69655e) component guidelines describing how Orange designers want cards to be represented and used.
```

Comment:
```suggestion
**On the Orange Design System website**, you'll find the [Cards component guidelines](https://system.design.orange.com/0c1af118d/p/82eaf1-cards/b/69655e) describing how Orange designers want cards to be represented and used.
```
For better a11y

Reply:
If I do that here, I will have to it in all the other design callout messages :-)